bindata/network/network-metrics/ipsec-rules.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
is this for sdn + ovn-k? if its just for ovn-k and you want to expose this for every deployment model we have, id put this in `bindata/network/ovn-kubernetes/common/alert-rules.yaml`. If its not meant for managed services and just non-managed standalone clusters, you can adjust where to put it.